charts/gateway/templates/node-properties-secret.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if not .Values.disklessConfig  }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name:  {{ template "nodePropertiesSecretName" . }}
```

Comment:
For nodePropertiesSecretName to get a value, need to the same thing done for gateway.license here:
https://github.com/CAAPIM/apim-charts/blob/node-properties/charts/gateway/templates/_helpers.tpl#L188